website/docs/r/data_protection_backup_vault.html.markdown

```diff
@@ -47,6 +47,10 @@ The following arguments are supported:
 
 * `identity` - (Optional) An `identity` block as defined below.
 
+* `soft_delete_setting` - (Optional) The state of soft delete for this Backup Vault. Possible values are `AlwaysOn`, `Off` and `On`. Defaults to `On`.
+
+-> **Note:** Once the `soft_delete_setting` is set to `AlwaysOn`, the setting cannot be changed.
```

Comment:
We should add a customize diff that forces a new resource to be created when changing this value from true to false

Reply:
Fixed.